pkg/cmd/experimental/buildchain/buildchain.go

```diff
@@ -309,19 +339,19 @@ func findRepoDeps(repo, tag string, all bool, buildConfigList []buildapi.BuildCo
 	var childNamespace, childName, childTag string
 	for _, cfg := range buildConfigList {
 		for _, tr := range cfg.Triggers {
+			from, fromTag := buildutil.GetImageStreamForStrategy(&cfg)
+			if from == nil {
```

Comment:
Mind adding `|| tr.ImageChange == nil` here and simplify all the following if cases?